### PR TITLE
Cover/cmds/core/df

### PR DESCRIPTION
**Pull request by ChriMarMe**

### CODE REVIEW

File: cmds/core/df/df.go [hunk 1]
```diff
@@ -116,31 +130,42 @@ func DiskUsage(mnt *Mount) {
...
 	mnt.PCT = uint8(math.Ceil(pct))
+	return nil
 }
 
 // SetUnits takes the command line flags and configures
 // the correct units used to calculate display values
-func SetUnits() {
-	if *inKB && *inMB {
-		log.Fatal("options -k and -m are mutually exclusive")
+func SetUnits(inKB, inMB bool) error {
```
[the remark this answers is not in the document]
Comment (ChriMarMe) [reply cmds/core/df/df.go:138]: Oh my.....

[the remark this answers is not in the document]
Comment (ChriMarMe) [reply cmds/core/df/df.go:138]: Someone 30 something years ago had a oddly skill of thinking about ways to torture future developers.

[the remark this answers is not in the document]
Comment (ChriMarMe) [reply cmds/core/df/df.go:138]: Ok, read the man page for df. The go implementation was not even close to the amount of args supported and not close to the way how it is done there. I will put this on the "Fun for later" list. This PR is "Take it, or leave it"

File: cmds/core/df/df.go [hunk 2]
```diff
@@ -116,31 +130,42 @@ func DiskUsage(mnt *Mount) {
...
 	} else {
 		units = KB
 	}
+	return nil
 }
 
-func df() {
-	SetUnits()
-	mounts, _ := mountinfo()
+func df(args []string, fargs flags, w io.Writer) error {
```
Comment (rminnich) [inline cmds/core/df/df.go:150]: I would still prefer to see, as in the other code, this:
df(w io.Writer, fargs flags, args ...string)

Comment (ChriMarMe) [reply cmds/core/df/df.go]: done

**Review by rminnich — APPROVED** [review]

**Pull request merged**